src/site/antora/modules/ROOT/pages/manual/migration.adoc

```diff
@@ -16,5 +16,247 @@ Licensed to the Apache Software Foundation (ASF) under one or more
+| _removed without replacement_ footnote:removal[]
+
+| {log4j2-url}/manual/appenders.html#KafkaAppender[Kafka appender]
+| _removed without replacement_ footnote:removal[]
+
+| xref:manual/lookups.adoc#KubernetesLookup[Kubernetes lookup]
+| external
+https://github.com/fabric8io/kubernetes-client/blob/main/doc/KubernetesLog4j.md[`kubernetes-log4j` maintained by Fabric8]
+
+| xref:manual/scripts.adoc[Scripting support]
+| {sonatype-url}/log4j-script[`log4j-script`]
+
+| {log4j2-url}/manual/webapp.html[Servlet application support: Java EE version]
+| _removed without replacement_ footnote:removal[]
+
+| {log4j2-url}/manual/appenders.html#SMTPAppender[SMTP appender: Java EE version]
+| _removed without replacement_ footnote:removal[]
+
+| {log4j2-url}/manual/lookups.html#_spring_boot_lookup[Spring Boot lookup]
+| integrated into
+https://central.sonatype.com/artifact/org.springframework.boot/spring-boot[`spring-boot`]
+version 3.0.0 or later
+
+|===
+
+[#system-properties]
+== Migrate configuration properties
+
+In order to support per-logger context configuration properties, the configuration properties sub-system of Log4j has been rewritten and the naming convention of properties have changed.
+To help you with the migration process a backward compatibility system has been implemented, which translates Log4j 2 property names to their Log4j 3 equivalents.
+
+If you use configuration properties to finely tune Log4j Core, make sure that:
+
+* either you use the naming convention from the
+xref:manual/systemproperties.adoc[Log4j 3 configuration properties]
+page (recommended),
+* or you use the naming convention from the
+{log4j2-url}/manual/systemproperties.html[Log4j 2 configuration properties]
+page.
+
+Most Log4j 3 configuration properties sources are **case-sensitive** and fuzzy matching is not applied to those sources.
+
+[#properties-configuration-file]
+== Java properties configuration file format
+
+The Java properties configuration file format in Log4j 3 has been rewritten to use
+https://github.com/FasterXML/jackson-dataformats-text/blob/2.18/properties/README.md[`jackson-dataformat-properties`]
+to convert Java properties into the native JSON configuration file format.
+See
+https://github.com/FasterXML/jackson-dataformats-text/blob/2.18/properties/README.md[Jackson documentation]
+for more details on the conversion.
+
+The only significant changes between the Log4j 2 and Log4j 3 properties format are:
+
+. As an alternative to using the `type` configuration attribute to specify the **plugin type** of a Log4j component, you can append the plugin type to the prefix of the parent component.
+For example, instead of:
++
+[source,properties]
+----
+appender.0.type = Console
+appender.0.name = CONSOLE
+appender.1.type = File
+appender.1.name = MAIN
+appender.1.fileName = logs/main.log
+appender.2.type = File
+appender.2.name = DEBUG_LOG
+appender.2.fileName = logs/debug.log
+----
++
+you can write:
```

Comment:
`can` or `must`?

Reply:
Its optional, although I would still use it in the examples, since it is much shorter (vertically).